wedlocks/src/templates/confirmEvent.js

```diff
@@ -1,9 +1,17 @@
+const ethWaterlooMessage = `
+
+After you check in to ETHWaterloo, we'll refund your stake in DAI by the end of the weekend. You'll be able to access those funds once you eject your Unlock account into a crypto wallet. We'll provide more details on that process soon! Note that this event ticket is non-transferable.`
+
 export default {
-  subject: () => 'You are attending an event!',
+  subject: params => `You've got your ticket for ${params.eventName}!`,
   text: params =>
     `Hello,
 
-This is just a reminder that you are attending the event ${params.eventName} on ${params.eventDate}!
+This is just a reminder that you are attending the event ${
+      params.eventName
+    } on ${params.eventDate}!${
+      params.eventName === 'ETHWaterloo' ? ethWaterlooMessage : ''
```

Comment:
How do you feel about doing a downcase comparison here to avoid any funny stuff?

Reply:
We could do that -- any thoughts on going by the lock address in the url instead of event name?

Reply:
sounds good, we'll probably need to double/triple test the method that does the extraction